.github/workflows/tests.yml

```
@@ -33,7 +33,7 @@ jobs:
       - name: Install Dependencies using pnpm
         run: pnpm install --no-frozen-lockfile && git checkout pnpm-lock.yaml
       - name: Build
-        run: pnpm run ts:check
+        run: pnpm run build
```

The name is wrong, not the command.
```suggestion
      - name: Check TypeScript
        run: pnpm run ts:check
```

---

We need to actually build the package for `pnpm bob check` to actually check the correctness of the ESM built version

---

https://guild-oss.slack.com/archives/CSFK5GGAU/p1740658884058789?thread_ts=1740658470.220519&cid=CSFK5GGAU

---

So `ts:check` is unused?

---

I suppose yes, since the build is already checking types too.

I use it locally while coding.